aircraftConfigs/A321_by_Glidi.sacfg

```diff
@@ -254,7 +254,7 @@
                 {
                     "name": "Flaps not in position",
                     "type": 2,
-                    "primaryCondition": "$sim/cockpit2/controls/flap_handle_deploy_ratio == 1 or\n$sim/cockpit2/controls/flap_handle_deploy_ratio == 0.75",
+                    "primaryCondition": "$sim/cockpit2/controls/flap_handle_deploy_ratio == 1 or $sim/cockpit2/controls/flap_handle_deploy_ratio >= 0.745",
```

Comment:
This is the main change, rest is typo and text cleanup.

We can increase this to 0.748 or 0.749 too if needed.

Reply:
I'm not yet entirely happy with that solution.

With your example, moving the flaps from 0.75 to 1 would be allowed below check height.
We'd have to make something like this:
`$sim/cockpit2/controls/flap_handle_deploy_ratio >= 0.95 or ($sim/cockpit2/controls/flap_handle_deploy_ratio >= 0.745 and $sim/cockpit2/controls/flap_handle_deploy_ratio <= 0.755)`
Which should be working, but in my opinion is getting quite complex for just a single flap position.

I'll have some spare time in the next few days, maybe I'll figure out a more elegant solution

Reply:
It's actually being checked from check height until reaching 60kn :)

Reply:
Yeah realized it while checking the end condition :)